sdk/src/asset_handlers/otf_io.rs

```diff
@@ -287,6 +287,8 @@ static SUPPORTED_TYPES: [&str; 10] = [
 const C2PA_TABLE_TAG: Tag = tables::C2PA::TAG;
 /// Tag for the 'head' table in a font.
 const HEAD_TABLE_TAG: Tag = tables::head::TAG;
+/// Lenght of the table directory header (i.e., before the table records)
```

Comment:
Lenghtttt!

Reply:
This was fixed in 1be2495, please review again.